msteams-ui-styles-core/src/colors.ts

```diff
@@ -16,12 +16,17 @@ export const Colors = {
     gray12: '#f4f4f4',
     gray13: '#F8F8F9', // don't know
     gray14: '#F9F8F7',
+    gary15: '#EDEBE9',
```

Comment:
gary is not a color =P

Reply:
It's a Typo Mistake. I did not save the file before commit. now resolved in the latest commit